brave/brave-ios
# Fix #8516: Add Open in Brave to iOS Share Sheet Action (Action Extension)

## Summary of Changes

This PR is enabling Brave Browser OS default Share Sheet for URL related actions. This will provide extra visibility and easy access to Brave users on iOS thorough other applications while using the Share Sheet.


This pull request fixes #8516

## Submitter Checklist:


## Test Plan:


- Open any app like Safari
- Highlight a text or a url
- Share
- Check Open in Brave action is there 
- Test using the open in brave actions for various types

## Screenshots:


https://github.com/brave/brave-ios/assets/6643505/b89d26ed-8143-439e-9d08-72d7349ae6e8


![Screenshot 2024-01-19 at 2 55 19 PM](https://github.com/brave/brave-ios/assets/6643505/ed06f859-74e3-4a26-ab63-e381141687a2)

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
